### PR TITLE
Automation repo template

### PR DESCRIPTION
**Pull request by alexderilo**

### CODE REVIEW

**Comment by papamai** [comment]
Thank you sa initiative na to fren @alexderilo <3.

Baka pwede palagay na rin nito:
- DB Utilities - establish and disconnect DB keywords
- error keywords - utilities and assertions
- gitignore

Tapos paclean-up din nung _pycache_ directories.

File: resources/keywords/db_assertions/common.robot
Comment (papamai) [inline resources/keywords/db_assertions/common.robot:1]: Xis, ito kahit isang keyword na lang lagay natin. HAHA

File: tests/acceptance/documents.child.merchant-get/test_authentication.robot [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Force Tags    regression    get_documents_child_org_onboarding    documents_child_org_onboarding
+# Add any setup or tear down keywords if applicable
+Test Setup    run keyword    prepare organization keys    SPLIT_ROBINSONS_BANK
+
+*** Test Cases ***
+# Sample test cases
+Request should return child organization's documents WHEN auth and child_organization_id is valid
+    GIVEN a child organization with all documents
+    WHEN get child merchant documents is called
+    THEN response status code should be    200
```
Comment (lorie143) [inline tests/acceptance/documents.child.merchant-get/test_authentication.robot:17]: Hello @alexderilo could you please check if we can add the following? thanks!
- add schema
- add assertions

File: resources/keywords/db_assertions/common.robot
Comment (alexderilo) [reply resources/keywords/db_assertions/common.robot:1]: oki na fren hahaha thanks

**Review by papamai — APPROVED** [review]

File: resources/keywords/assertions/common.robot [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+object livemode should be
+    [Arguments]    ${variable}
+    ${livemode}=    get value from response via json path   $.data.attributes.livemode
+    ${boolean}    convert to string    ${livemode}
+    run keyword if    ${variable} == True    should be equal    ${boolean}    True
+    run keyword if    ${variable} == False    should be equal    ${boolean}    False
+
+response field length should be
+	[Arguments]    ${json_path}    ${expected_value}
```
Comment (lorie143) [inline resources/keywords/assertions/common.robot:38]: fixed spacing

File: resources/keywords/requests/core.organizations.id-get/request.robot [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Resource    ../../libraries.robot
+Resource    ../../../tests/resource_collection.robot
+
+*** Keywords ***
+# Sample request keyword
+get organization is requested
+    [Arguments]    ${auth}=${${env}_LIVE_SUPERUSER}    ${organization_id}=${organization_id}
+    ${auth}=    create list    ${auth}    ""
+    @{query_param_list}    create list
+	${query_parameters}    set variable    ${EMPTY}
```
Comment (lorie143) [inline resources/keywords/requests/core.organizations.id-get/request.robot:14]: fix spacing

File: resources/keywords/requests/core.organizations.id-get/request.robot [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    ${headers}    create dictionary    Authorization=Basic
+    run keyword if    "${organization_id}" != "None"    append to list    ${query_param_list}    ${organization_id}
+	FOR    ${param}    IN    @{query_param_list}
```
Comment (lorie143) [inline resources/keywords/requests/core.organizations.id-get/request.robot:17]: fix spacing

File: resources/keywords/requests/core.organizations.id-get/request.robot [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+		${query_parameters}    set variable if    "${query_parameters}" == "${EMPTY}"    ${param}
+	END
+    create session    session    ${${env}_CORE_HOST}    auth=${auth}    disable_warnings=1
+    ${response}    get on session    session    url=/v1/organizations/${query_parameters}    headers=${headers}    expected_status=any
+    set test variable    ${response}
+	log    ${response}
```
Comment (lorie143) [inline resources/keywords/requests/core.organizations.id-get/request.robot:23]: fix spacing

File: resources/keywords/utilities/db_utilities.robot [hunk 6]
```diff
@@ -0,0 +1,14 @@
+# Below are the commonly used database utilities keywords
+
+*** Keywords ***
+establish core database connection
+    decrypt superuser keys
+	${DB_PASSWORD}    get decrypted text    ${${env}_DB_PASSWORD}
```
Comment (lorie143) [inline resources/keywords/utilities/db_utilities.robot:6]: fix spacing

**Review by seruuh — APPROVED** [review]

**Review by lorie143 — CHANGES_REQUESTED** [review]
Hello @alexderilo can we also add the requests folder? thanks!

**Review by lorie143 — APPROVED** [review]
Approved on my end, thanks!

**Review by marc-rint-paymongo — APPROVED** [review]

**Review by GOlmillaPM — APPROVED** [review]
Looks good :) Thank you!

**Pull request merged**